internal/pkg/describe/env.go

```diff
@@ -317,6 +317,39 @@ func (d *EnvDescriber) filterDeployedJobs() ([]*config.Workload, error) {
 	return deployedJobs, nil
 }
 
+// ValidateCFServiceDomainAliases returns error if an environment using cdn is deployed without specifying http.alias for all services
+func (d *EnvDescriber) ValidateCFServiceDomainAliases() error {
+
+	stackDescr, err := d.cfn.Describe()
+	if err != nil {
+		return err
+	}
+
+	services, err := d.deployStore.ListDeployedServices(d.app, d.env.Name)
+	if err != nil {
+		return fmt.Errorf("list services: %w", err)
+	}
+
+	if jsonOutput, ok := stackDescr.Parameters[envParamAliases]; ok {
+		var aliases map[string][]string
+		err := json.Unmarshal([]byte(jsonOutput), &aliases)
+
+		if err != nil {
+			return fmt.Errorf("failed to unmarshal \"%s\": %w", jsonOutput, err)
+		}
+
+		for _, service := range services {
+			if _, ok := aliases[service]; !ok {
+				return fmt.Errorf("all services deployed in an environment with CloudFront enabled must have http.alias specified")
+			}
+		}
+	} else {
+		return fmt.Errorf("all services deployed in an environment with CloudFront enabled must have http.alias specified")
```

Comment:
If it is not ok to get the parameter we might want to throw a diffferent error? Like cannot find xxx in the env stack parameter set.